update_multiarch.sh

```diff
@@ -71,7 +71,7 @@ do
 				if [ "${vm}" != "hotspot" ]; then
 					reldir="${reldir}-${vm}";
 				fi
-				generate_dockerfile ${file} ${build} ${btype} ${os}
+				generate_dockerfile.sh ${file} ${build} ${btype} ${os}
```

Comment:
`generate_dockerfile` is bash function in `dockerfile_function.sh`, why you have added `.sh` ?